package.json

```diff
@@ -152,6 +152,12 @@
             "Open",
             "Add to Workspace"
           ]
+        },
+        "spring.initializr.createArtifactIdFolder": {
```

Comment:
I'm not sure if `createArtifactIdFolder` is a perfect name. A suggestion here is to make this configuration an Enum value:

- spring.initializr.parentFolder: "artifactId" || "none"

"artifactId" means to create a new folder named after artifact id and "none" means to put the project in the current folder. The advantage is we can keep the scalability of this setting.

Reply:
I changed that name a couple times, found it hard to avoid creating a very complex one.
The idea of using an enum seems much better, both for maintainability and UX when looking for the setting.

I can do this change later today, after work.

Reply:
enum has more extensibility. Agree to use this enum, in case later someone else wants to use `name` property as parent folder name.